data/benefits/additional-attributes/index.ts

```diff
@@ -65,6 +65,15 @@ export const additionalBenefitAttributes = {
       },
     },
   },
+  "cohesion-territoires-conseillers-numeriques-france-services": {
```

Comment:
y'a une raison pour utiliser les tirets et pas le underscore comme les autres clefs ?

Reply:
Les clés de cet objet correspondent aux slugs, ie. aux noms des fichiers des aides associées. La règle de lint implémentée pour les noms de fichier est le [kebab-case](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841).